### DISCUSSION TITLE
Configuration settings location

### DISCUSSION THREAD

**Discussion by kevinoptimezers** [Romanitho/Winget-AutoUpdate#635] [Q&A] [answered]
Where are the configuration settings stored (when not using GPO's)? 

We deploy the application through Intune and would like to, next to detecting the application, detect if the correct whitelist is used.

**Comment by Romanitho** [comment, marked as answer]
HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\Winget-AutoUpdate

whitelist is in winget-autoupdate folder under programdata
